edx_toggles/toggles/internal/waffles.py

```diff
@@ -0,0 +1,132 @@
+"""
+Waffle classes in the context of edx-platform and other IDAs.
```

Comment:
Nit: I'd prefer `waffle.py` for the name of the file.

Reply:
:ok_hand: